tensorflow_addons/layers/wrappers.py

```diff
@@ -224,3 +224,112 @@ def remove(self):
             self.layer.kernel = kernel
```

Comment:
nit: Please update the copyright to 2020

Reply:
Done